raiden/utils/filters.py

```diff
@@ -103,19 +112,43 @@ def __init__(self, web3: Web3, filter_params: dict):
         self._last_block: int = -1
         self._lock = Semaphore()
 
-    def get_new_entries(self, block_number: BlockSpecification):
+    def _do_get_new_entries(self, from_block: BlockSpecification, to_block: BlockSpecification):
+        filter_params = self.filter_params.copy()
+        filter_params['fromBlock'] = from_block
+        filter_params['toBlock'] = to_block
+
+        log.debug(f'Querying filter from block {from_block} to block {to_block}')
```

Comment:
This logging entry can go or it can stay. Not sure about it. Helped during debugging, but will spam the screen with the query batches the first time raiden starts.

Reply:
I think it's reasnoable to have it, but I'd put `from_block`  and `to_block`  as kwargs.

Reply:
Paul asks, Paul gets: https://github.com/raiden-network/raiden/pull/3617/commits/278bec8ad1ec032634506f9822aceae01d842b40